the-collab-lab/tcl-23-smart-shopping-list
# updated favicon and app title

## Description

I've updated the favicon image and app title in `public/index.html `to reflect the app's name and theme.  

## Related Issue

This is part of a series of sub-issues related to #61 

## Acceptance Criteria

- [x] Update link tag to `<link rel="icon" href="%PUBLIC_URL%/favicon.ico?v=2" />`
- [x] Update app title to `<title>HoneyDo</title>`
- [x] Add new favicon

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|   | :sparkles: New feature     |
|  ✓ | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

<img width="1366" alt="Screen Shot 2021-05-29 at 7 01 30 PM" src="https://user-images.githubusercontent.com/23660204/120086824-4afffe80-c0b0-11eb-9d88-fec9d20c892f.png">



### After

<img width="1372" alt="Screen Shot 2021-05-29 at 6 37 22 PM" src="https://user-images.githubusercontent.com/23660204/120086746-a67dbc80-c0af-11eb-8333-1701be7bcc39.png">



## Testing Steps / QA Criteria

1. Pull branch za-add-flavicon
2.  Use npm start command to run the app in your local environment
3. App should run with correct favicon and title in the browser's active tab
